src/test/java/org/fogbowcloud/manager/core/cloudconnector/LocalCloudConnectorTest.java

```diff
@@ -90,577 +84,847 @@
-	    when(this.order.getType()).thenReturn(InstanceType.NETWORK);
-	    when(networkPlugin.requestInstance(any(NetworkOrder.class), any(Token.class))).thenReturn(FAKE_INSTANCE_ID);
-		assertEquals(FAKE_INSTANCE_ID, this.localCloudConnector.requestInstance(order));
-		Mockito.verify(computePlugin, times(0)).requestInstance(any(ComputeOrder.class), any(Token.class));
-	    Mockito.verify(volumePlugin, times(0)).requestInstance(any(VolumeOrder.class), any(Token.class));
-	    Mockito.verify(attachmentPlugin, times(0)).requestInstance(any(AttachmentOrder.class), any(Token.class));
-	    Mockito.verify(networkPlugin, times(1)).requestInstance(any(NetworkOrder.class), any(Token.class));
+	    Mockito.when(this.order.getType()).thenReturn(InstanceType.NETWORK);
+	    Mockito.when(networkPlugin.requestInstance(Mockito.any(NetworkOrder.class), Mockito.any(Token.class))).thenReturn(FAKE_INSTANCE_ID);
+	    
+	    //exercise
+	    String returnedInstanceId = this.localCloudConnector.requestInstance(order);
+	    
+	    // verify
+	    Assert.assertEquals(FAKE_INSTANCE_ID, returnedInstanceId);
+		Mockito.verify(computePlugin, times(0)).requestInstance(Mockito.any(ComputeOrder.class), Mockito.any(Token.class));
+	    Mockito.verify(volumePlugin, times(0)).requestInstance(Mockito.any(VolumeOrder.class), Mockito.any(Token.class));
+	    Mockito.verify(attachmentPlugin, times(0)).requestInstance(Mockito.any(AttachmentOrder.class), Mockito.any(Token.class));
+	    Mockito.verify(networkPlugin, times(1)).requestInstance(Mockito.any(NetworkOrder.class), Mockito.any(Token.class));
 	}
 	
-	// If plugin returns a null instance id, the method requestInstance() must throw an exception
+	// test case: If plugin returns a null instance id, the method requestInstance() must throw an exception
 	@Test(expected = UnexpectedException.class)
 	public void testExceptionNullComputeInstanceId() throws FogbowManagerException, UnexpectedException {
+		
+		// set up
 		this.order = Mockito.mock(ComputeOrder.class);
-	    when(this.order.getType()).thenReturn(InstanceType.COMPUTE);
-	    when(computePlugin.requestInstance(any(ComputeOrder.class), any(Token.class))).thenReturn(null);
-		this.localCloudConnector.requestInstance(order);
-		Mockito.verify(computePlugin, times(1)).requestInstance(any(ComputeOrder.class), any(Token.class));
-	    Mockito.verify(volumePlugin, times(0)).requestInstance(any(VolumeOrder.class), any(Token.class));
-	    Mockito.verify(attachmentPlugin, times(0)).requestInstance(any(AttachmentOrder.class), any(Token.class));
-	    Mockito.verify(networkPlugin, times(0)).requestInstance(any(NetworkOrder.class), any(Token.class));
+	    Mockito.when(this.order.getType()).thenReturn(InstanceType.COMPUTE);
+	    Mockito.when(computePlugin.requestInstance(Mockito.any(ComputeOrder.class), Mockito.any(Token.class))).thenReturn(null);
+		
+		// exercise
+	    this.localCloudConnector.requestInstance(order);
+	    
+	    // verify
```

Comment:
Since we expect line 231 throws an exception, below line are not executed. remove them.